dqc_us_rules/dqc_us_0004.py

```diff
@@ -41,9 +40,19 @@ def _assets_eq_liability_equity(modelXbrl):
                 dec_assets = inferredDecimals(fact_assets)
                 dec_liabilities = inferredDecimals(fact_liabilities)
                 min_dec = min(dec_assets, dec_liabilities)
-                if _values_unequal(fact_assets.xValue, fact_liabilities.xValue, min_dec):
+                if _min_dec_valid(min_dec) and _values_unequal(fact_assets.xValue, fact_liabilities.xValue, min_dec):
                     yield fact_assets, fact_liabilities
 
+def _min_dec_valid(min_dec):
+    """
+    Checks to make sure that min_dec values are valid.
+    Returns False if min_dec is a None type or if min_dec is Not a number
+
+    @param min_dec:
```

Comment:
Also the format is the wrong type, poke me tomorrow and I will fix your pycharm for you